packages/fx-core/src/component/driver/aad/utility/createAuthFiles.ts

```diff
@@ -5,11 +5,11 @@ import { err, FxError, Inputs, ok, Result, SystemError } from "@microsoft/teamsf
 import AdmZip from "adm-zip";
 import fs from "fs-extra";
 import path from "path";
-import { getLocalizedString } from "../../../common/localizeUtils";
-import { unzip } from "../../generator/utils";
-import { FileNotFoundError } from "../../../error/common";
-import { getTemplatesFolder } from "../../../folder";
-import { AddSsoParameters, SolutionError, SolutionSource } from "../../constants";
+import { getLocalizedString } from "../../../../common/localizeUtils";
```

Comment:
I personally prefer to put this file to `feature` folder (along with sso.ts), since it's not related to AAD's CRUD operation.